index.js

```diff
@@ -1 +1,135 @@
 console.log("Hello!!!");
+
+//Prototype inheritance
+
+// Create a prototype chain Inherit from university obj with universityName and dean properties
+// Create faculty obj with facultyName and groups arr properties and method enlistStudent into groups
+// Group can contain only 12 students
+
+const university = {
+    universityName: "Polytechnic",
+    dean: "Vladimir Vladimirov",
+};
+  
+const faculty = {
+    __proto__: university,
+    facultyName: "psychology",
+    groups: [],
+    enlistStudent(student) {
+        if (this.groups.length < 12){
```

Comment:
Prototype inheritance is done ok 
but logic of this method has a bug
double check expected result and your result